stacks/elb_ssl.json

```diff
@@ -0,0 +1,14 @@
+{
+  "SSLCertificateId" : {
+    "Fn::Join":[
+       "",
+       [
+          "arn:aws:iam::",
+          {
+             "Ref":"AWS::AccountId"
+          },
+          ":server-certificate/"
+       ]
+    ]
+  }
+}
```

Comment:
This file is in the wrong place - you've updated the code to use pkgutil.get_data but not moved the file to live at awsutils/stacks/elb_ssl.json